mrSerious/StackOverflow-lite
# #160358024 Consumes get all questions endpoint

#### What does this PR do?
Have a front-end that consume the get all questions endpoint
#### Description of Task to be completed?
Have a front-end to consume the following endpoint.
`GET /questions`
#### How should this be manually tested?
After cloning the repo, cd into the folder and run `npm install`
Check that front-end consumes the endpoint as follows:
- Open your browser
- Navigate to `https://stack-overflow-lite-app.herokuapp.com/api/v1/questions`
- You should be presented the home page displaying a list of recently asked questions.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#160358024](https://www.pivotaltracker.com/n/projects/2189147/stories/160358024)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
